### ISSUE TITLE
Blob.find(user,repo,sha) Fails Due to Invalid Content-Type

### ISSUE THREAD

**Issue by runpaint** [fcoury/octopi#12]
The GitHub API returns the wrong Content-Type so this call raises an Exception. This is blocked by develop/develop.github.com#13 ; if that gets wontfix'd, we'll need a workaround.

**Comment by radar** [comment]
This is working now, to my knowledge.
